python/ml_croissant/ml_croissant/_src/operation_graph/operations/extract_test.py

```diff
@@ -5,5 +5,5 @@
 
 
 def test_str_representation():
```

Comment:
I'm not familiar with the codebase, but does this mean that we are testing only with empty files? Wouldn't it make sense at some point to also test w. dummy non-empty file(set)s?

Reply:
In this PR, I add datasets/coco2014-mini/data/train2014.zip. So we do test on non empty files.

This test is for the string representation.